example-conf/mesosphere-dcos/mesos-site.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<!--
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+
+<!-- Put site-specific property overrides in this file. -->
+
+<configuration>
+  <property>
+    <name>mesos.hdfs.framework.mnt.path</name>
+    <value>/opt/mesosphere</value>
+    <description>This is the default for all DCOS installs</description>
+  </property>
+
+  <property>
+    <name>mesos.hdfs.state.zk</name>
+    <value>master.mesos:2181</value>
+    <description>See the Mesos DNS config file for explanation for this</description>
+  </property>
+
+  <property>
+    <name>mesos.master.uri</name>
+    <value>zk://master.mesos:2181/mesos</value>
+    <description>See the Mesos DNS config file for explanation for this</description>
+  </property>
+
+  <property>
+    <name>mesos.hdfs.zkfc.ha.zookeeper.quorum</name>
+    <value>master0.mesos:2181,master1.mesos:2181,master2.mesos:2181</value>
+    <description>See the Mesos DNS config file for explanation for this</description>
+  </property>
+
+  <property>
+    <name>mesos.hdfs.mesosdns</name>
+    <value>true</value>
+    <description>All DCOS installs come with mesos DNS to maintain static configurations</description>
+  </property>
+
+  <property>
+    <name>mesos.hdfs.preshared-hadoop-binaries</name>
+    <value>true</value>
+    <description>DCOS comes with a pre-shared HDFS configuration in a single-tenant environment</description>
```

Comment:
I meant that "preshared" in the 'name' and "pre-shared" in the description should both be changed to "native" now that you've renamed the rest.